### ISSUE TITLE
build Cxx makes an error in Julia v1.6.1

### ISSUE THREAD

**Issue by andyname** [JuliaInterop/Cxx.jl#489]
After Pkg.add ("Cxx"), if I type "build Cxx" in pkg mode results in the following error. I'm currently using the latest version of Julia v1.6.1 on windows 10. Do I have to wait longer to use this package on Windows?

(@v1.6) pkg> build Cxx
Building Cxx → `C:\Users\andyn\.julia\scratchspaces\44cfe95a-1eb2-52ea-b672-e2afdf69b78f\71728149f62225470a4ae4260536a98a5009997c\build.log`
ERROR: Error building `Cxx`:
[ Info: Building julia binary build
┌ Warning: Windows support is still experimental!
└ @ Main C:\Users\andyn\.julia\packages\Cxx\1RaOv\deps\build.jl:37
ERROR: LoadError: LoadError: "C:\\Users\\andyn\\AppData\\Local\\Programs\\Julia-1.6.0\\share\\julia\\stdlib\\v1.6\\BinaryProvider\\JuliaTeam.toml": No such file
Stacktrace:
  [1] error(::String, ::String)
    @ Base .\error.jl:42
  [2] readstring
    @ C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.6\TOML\src\TOML.jl:21 [inlined]
  [3] parsefile(f::String)
    @ TOML C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.6\TOML\src\TOML.jl:43
  [4] redirect_download_target(url::String)
    @ BinaryProvider.JuliaTeam ~\.julia\packages\BinaryProvider\IjWSK\src\JuliaTeam.jl:35
  [5] (::BinaryProvider.JuliaTeam.var"#1#2"{BinaryProvider.var"#56#85"{BinaryProvider.var"#51#80"{Cmd, String}}})(url::String, file::String)
    @ BinaryProvider.JuliaTeam ~\.julia\packages\BinaryProvider\IjWSK\src\JuliaTeam.jl:18
  [6] download(url::String, dest::String; verbose::Bool)
    @ BinaryProvider ~\.julia\packages\BinaryProvider\IjWSK\src\PlatformEngines.jl:608
  [7] download_verify(url::String, hash::String, dest::String; verbose::Bool, force::Bool, quiet_download::Bool)
    @ BinaryProvider ~\.julia\packages\BinaryProvider\IjWSK\src\PlatformEngines.jl:692
  [8] install(tarball_url::String, hash::String; prefix::Prefix, tarball_path::String, force::Bool, ignore_platform::Bool, verbose::Bool)
    @ BinaryProvider ~\.julia\packages\BinaryProvider\IjWSK\src\Prefix.jl:314
  [9] top-level scope
    @ ~\.julia\packages\Cxx\1RaOv\deps\build_libcxxffi.jl:35
 [10] include(fname::String)
    @ Base.MainInclude .\client.jl:444
 [11] top-level scope
    @ ~\.julia\packages\Cxx\1RaOv\deps\build.jl:40
 [12] include(fname::String)
    @ Base.MainInclude .\client.jl:444
 [13] top-level scope
    @ none:5
in expression starting at C:\Users\andyn\.julia\packages\Cxx\1RaOv\deps\build_libcxxffi.jl:33
in expression starting at C:\Users\andyn\.julia\packages\Cxx\1RaOv\deps\build.jl:28
writing path.jl file
Tuning for julia installation at C:\Users\andyn\AppData\Local\Programs\Julia-1.6.0\bin with sources possibly at C:\Users\andyn\AppData\Local\Programs\

**Comment by Gnimuc** [comment]
> Do I have to wait longer to use this package on Windows?

Yes. AFAICT, this package has never hit a stable release on Windows. 

BTW, the Julia version supported by this package is 1.1~1.3.

**Comment by andyname** [comment]
Thanks a lot. I should use PyCall instead.
